package.json

```diff
@@ -55,6 +55,7 @@
     "babel-core": "^6.22.1",
     "babel-loader": "^6.2.10",
     "babel-plugin-istanbul": "^4.1.1",
+    "babel-plugin-remove-comments": "^2.0.0",
```

Comment:
We can get rid of this plugin and get the same behavior by setting `"comments": "false"` in our babel config.